tests/e2e/specs/plugin-activation.test.js

```diff
@@ -48,6 +48,7 @@ describe( 'Plugin Activation Notice', () => {
 		await activateSiteKit();
 
 		await page.waitForSelector( '.googlesitekit-activation' );
+		await page.waitForSelector( 'h3.googlesitekit-activation__title' );
```

Comment:
Let's remove the `h3` from this selector (and below) and rely on the class only. The specific element isn't really important and will make the test fail unnecessarily should that change in the future.

Reply:
Good shout! Actually, we can probably remove the `. googlesitekit-activation` waitFor as well, so I'll do that at the same time.